src/js/components/NewPageHeaderTabs.js

```diff
@@ -10,7 +10,7 @@ class PageHeaderTabs extends React.Component {
       let {isActive} = tab;
       let classes = classNames('tab-item', {active: isActive});
       let linkClasses = classNames('tab-item-label', {active: isActive});
-
+      // TODO react-router: routeName ??? Should it be routePath?
```

Comment:
Change `routeName` to `routePath` since nobody is currently using this component, just make sure we keep this change somewhere. @jfurrow will create an issue, add a comment there.